README.md

```diff
@@ -179,21 +179,10 @@ private let foragePinTextField: ForagePINTextField = {
     let tf = ForagePINTextField()
     tf.borderRadius = 10
     tf.backgroundColor = .systemGray6
-    tf.pinType = .balance
     return tf
 }()
 ```
 
-To identify the type of pin we are handling in the component, you can use the `pinType` property. We have support for these types:
-
-```swift
-public enum PinType: String {
-    case snap
-    case nonSnap
-    case balance
-}
```

Comment:
If we wanted to gracefully deprecate this without having to release a new major version, we could use the `@available` feature

```swift
@available(*, deprecated, message: "PinType is deprecated.")
public enum PinType: String {
```

Reply:
I don't think this is used by anybody and may just rip the silent code removal